docs/reference/asyncapi-rules.md

```diff
@@ -0,0 +1,311 @@
+# AsyncAPI Rules
+
+Spectral has a built-in "asyncapi" ruleset for the [AsyncAPI Specification](https://www.asyncapi.com/docs/specifications/2.0.0/).
+
+In your ruleset file you can add `extends: "spectral:asyncapi"` and you'll get all of the following rules applied.
+
+## AsyncAPI v2-only
```

Comment:
```suggestion
```

we only had this in here when we were documenting both oas2 and oas3 at the same time.

Reply:
@philsturgeon Fixed